parser/parser_test.go

```diff
@@ -1467,7 +1467,8 @@ func (s *testParserSuite) TestDDL(c *C) {
 		{"CREATE TABLE IF NOT EXISTS `general_log` (`event_time` timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6),`user_host` mediumtext NOT NULL,`thread_id` bigint(20) unsigned NOT NULL,`server_id` int(10) unsigned NOT NULL,`command_type` varchar(64) NOT NULL,`argument` mediumblob NOT NULL) ENGINE=CSV DEFAULT CHARSET=utf8 COMMENT='General log'", true},
 
 		// for alter table
-		{"ALTER TABLE t ADD COLUMN( a SMALLINT UNSIGNED )", true},
+		{"ALTER TABLE t ADD COLUMN (a SMALLINT UNSIGNED)", true},
+		{"ALTER TABLE t ADD COLUMN (a SMALLINT UNSIGNED FIRST)", false},
```

Comment:
It seems this DDL statement not valid for MySQL. 

```
mysql> ALTER TABLE s1 ADD COLUMN (a SMALLINT UNSIGNED FIRST);
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'FIRST)' at line 1
```
> Server version: 5.7.18-log MySQL Community Server (GPL)

Reply:
Yes, so the result is `false`.

Reply:
Oh, sorry for my carelessness.